lib/CMakeLists.txt

```diff
@@ -296,6 +298,7 @@ endif()
 
 # Link in additional libraries
 target_link_libraries(libfacter PRIVATE
+    ${CPPHOCON_LIBRARIES}
```

Comment:
Yes it will, though I think you're right, it's not actually needed for this PR (not sure how it slipped in). Happy to remove it if you'd like.